dppy/exotic_dpps.py

```diff
@@ -466,6 +464,12 @@ class UST:
     """
 
     def __init__(self, graph):
+        # For Uniform Spanning Trees
+        try:
+            import networkx as nx
+            self.nx = nx
```

Comment:
Hi @danielecc,
Why the additional `self.nx`?

Reply:
nx is used in multiple places in the class (eg [here](https://github.com/guilgautier/DPPy/pull/60/files/dd342cb165dcc93c2978cca2f0141cf8497acbc4#diff-fc9c2f2c4d646ae92e8cf2184fe5fc63R574)). Before it was imported at a global level so it was ok, but now that it's a local import it must be stored somewhere for following accesses.

Reply:
All right, otherwise methods of the class don't have access to `nx`